Sensors@claudiux/files/Sensors@claudiux/applet.js

```diff
@@ -395,11 +396,13 @@ class SensorsApplet extends Applet.TextApplet {
         this.strictly_positive_fan ? 1 : 0,
         this.strictly_positive_volt ? 1 : 0
       );
+      this.reaper.reap_nvidia_smi();
     } else {
       this.set_applet_label(_("Suspended"));
     }
 
     this.loopId = Mainloop.timeout_add(this.interval * 1000, () => this.reap_sensors());
+    this.nvidiaLoopId = Mainloop.timeout_add(this.interval * 1000, () => this.reap_nvidia_smi());
```

Comment:
The line 405 seems useless and contains an error.
`this.reap_nvidia_smi()` sould be `this.reaper.reap_nvidia_smi()` and a loop already exists calling `this.reap_sensors()` which calls `this.reaper.reap_nvidia_smi()`.

Please remove this.nvidiaLoopId from this applet.